pkgs/by-name/by/bytecode-viewer/package.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, stdenv
+, fetchurl
+, jdk8
+, makeWrapper
+}:
+
+stdenv.mkDerivation (finalAttrs: {
```

Since this doesn't need a C/C++ compiler for anything
```suggestion
stdenvNoCC.mkDerivation (finalAttrs: {
```